src/main/resources/liquibase/crop_changelog/22_4_0.xml

```diff
@@ -87,4 +87,18 @@
 			VALUES ('gigwa_password', 'Password for specified crop genotyping username', 1, 'gigwa');
 		</sql>
 	</changeSet>
+
+	<changeSet author="cuenyad" id="v22.4.0-6">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				select count(1) FROM program_preset where configuration like '%VARIABLE%' or '%STATIC%'
```

Comment:
@cuenyad no way that VARIABLE or STATIC are present i the configuration before this change?

Reply:
@clarysabel before this change only were stored ids.

Reply:
ok, Id of variables and names

Reply:
Id of variables and names with the  name id increased to 10000, and static